### PR TITLE
 Fix YQTree's icon loading function (boo#1116562) 

### PR DESCRIPTION
**Pull request by hellcp**

### CODE REVIEW

File: src/YQTree.cc [hunk 1]
```diff
@@ -492,7 +493,10 @@ void YQTreeItem::init( YQTree *		tree,
 
     if ( _origItem->hasIconName() )
     {
-        QIcon icon = YQUI::ui()->loadIcon( _origItem->iconName() );
+        //YQTree is annoying with how it loads icons, this is an ugly walkaround
```
Comment (shundhammer) [inline src/YQTree.cc]: workaround, I hope ;-)

Comment (hellcp) [reply src/YQTree.cc]: Gremmer :P

File: src/YQWizard.cc [hunk 2]
```diff
@@ -774,11 +774,13 @@ QWidget *YQWizard::layoutWorkArea( QWidget * parent )
     //headingHBox->setSizePolicy( QSizePolicy( QSizePolicy::Expanding, QSizePolicy::Minimum ) ); // hor/vert
     leftInnerBox->addLayout( headingHBox );
 
+    /*
```
Comment (shundhammer) [inline src/YQWizard.cc]: Please either use `#if 0 .... #endif` or simply delete that block; it is too easily overlooked that this is commented out. This very much depends on the editor that people use and how good that editor's syntax highlighting is.

Comment (hellcp) [reply src/YQWizard.cc]: Noted

**Comment by shundhammer** [comment]
LGTM

**Pull request merged**